app/views/lookup_keys/_fields.html.erb

```diff
@@ -31,9 +31,8 @@
   <fieldset>
     <%= collapsing_legend _("Optional input validator"), "#optional_input_validators_#{f.object.id}", "collapsed" %>
     <div id="optional_input_validators_<%= f.object.id %>" class="collapse out">
-      <p class="help-block">
-        <%= _('Note that if you use ERB as a value of parameter, value will be validated during ENC evaluation. If value is invalid, ENC evaluation will fail.') %>
-      </p>
+      <%= alert :class => 'alert-info', :header => 'Note', :text => _('If you use ERB as a value of parameter, value will be validated during ENC evaluation. If value is invalid, ENC evaluation will fail.') %>
```

Comment:
Can you change this to `If ERB is used in a parameter value, the validation of the value will happen during the ENC request. If the value is invalid, the ENC request will fail.` as @domcleal mentioned on #theforeman-dev? Maybe remove the Note header, or if you want to leave it there, extract the string with _()